dist/index.js

```diff
@@ -7266,7 +7266,7 @@ function isDirectory(fsPath, useStat = false) {
 }
 exports.isDirectory = isDirectory;
 /**
- * On OSX/Linux, true if path starts with '/'. On Windows, true for paths like:
```

Comment:
This appears to be from: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/packages/io/src/io-util.ts#L44